ert3/workspace/_workspace.py

```diff
@@ -126,3 +128,14 @@ def initialize(path: Union[str, Path]) -> Workspace:
         )
     (path / _WORKSPACE_DATA_ROOT).mkdir()
     return Workspace(path)
+
+
+def _validate_inputs(
+    stage_config: ert3.config.StagesConfig, ensemble_config: ert3.config.EnsembleConfig
+) -> None:
+    ensemble_input_names = set(input.record for input in ensemble_config.input)
+    first_stage_input_names = set(stage_config[0].input.keys())
```

Comment:
Should this not be evaluated against the stage that has the same name as the one specified in the forward_model in the ensemble config, and not the first in the list? (Perhaps a bit related to https://github.com/equinor/ert/issues/2327)

Reply:
Definitely, thanks.